test/src/module/command/infoTest.c

```diff
@@ -652,7 +652,9 @@ testRun(void)
             "\"db-id\":1,\"option-archive-check\":true,\"option-archive-copy\":false,\"option-backup-standby\":false,"
             "\"option-checksum-page\":true,\"option-compress\":true,\"option-hardlink\":false,\"option-online\":true}\n"
             "20181119-152138F_20181119-152155I={"
-            "\"backrest-format\":5,\"backrest-version\":\"2.08dev\",\"backup-archive-start\":\"000000010000000000000003\","
+            "\"backrest-format\":5,\"backrest-version\":\"2.08dev\","
+            "\"backup-annotation\":{\"key1\":null},"
```

Comment:
How is it possible to have a NULL key? Don't we remove any keys the user specifies as empty?

Reply:
In theory we shouldn't have null values indeed, but the test is there for the coverage of `src/command/info/info.c` L976:
```
if (value != NULL)
    strCatFmt(annotationStr, "                %s: %s\n", strZ(key), strZ(value));
```

Reply:
If there are NULLs in the annotation data then we have a bug somewhere, right? Seems like this should be an error?

Reply:
If NULL always mean empty in our case, indeed that would mean we missed something somewhere. Would it worth changing `if` to `ASSERT` and fail the entire command rather than simply ignoring it?

Reply:
I played again with the current implementation and couldn't generate null "value". But just in case I switched the if to assert as discussed here in 0964d74ac1d885e663a9e85189c08bea42543d2e.